config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/EntityBundleLoader.java

```diff
@@ -4,12 +4,9 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-package com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle;
+package com.ca.apim.gateway.cagatewayconfig.bundle.loader;
 
-import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Bundle;
-import com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle.loader.BundleDependencyLoader;
-import com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle.loader.DependencyBundleLoadException;
-import com.ca.apim.gateway.cagatewayconfig.tasks.zip.bundle.loader.DependencyLoaderRegistry;
+import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary import: same package classes are always implicitly imported. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=122&issues=AWblzewmSgN4uCclhZKc&open=AWblzewmSgN4uCclhZKc)

[ISSUE_KEY]: AWblzewmSgN4uCclhZKc